server/routes/api/place.py

```diff
@@ -459,3 +462,61 @@ def api_ranking(dcid):
             result[label] = [x for x in result[label] if 'data' in x]
     result['label'] = [x for x in all_labels if x in result]
     return Response(json.dumps(result), 200, mimetype='application/json')
+
+
+@cache.memoize(timeout=3600 * 24) # Cache for one day.
+def get_state_code(dcid):
+    """Get state code for places that have a US State as one of its parent places
+
+    Args:
+        dcid: dcid of place we are looking for the state code for
+
+    Returns:
+        state code as a string or None if there is no US state code
+    """
+    state_code = None
+    for parent_place in json.loads(parent_places(dcid)):
+        parent_dcid = parent_place['dcid']
+        place_types = parent_place['types']
+        for place_type in place_types:
+            if place_type in STATE_EQUIVALENTS:
+                iso_code = get_property_value(parent_dcid, 'isoCode')
```

Comment:
@shanth can we rely on isoCode to fetch state codes? At the moment, using this for US places only but would be great to expand internationally too.

Reply:
@pradh

Reply:
Sorry I missed this. Using isoCode sounds good and is reasonable.